django/cantusdb_project/main_app/signals.py

```diff
@@ -201,3 +205,64 @@ def update_prefix_field(instance):
         instance.__class__.objects.filter(pk=pk).update(prefix=prefix)
     else:  # feast_code is None, ""
         instance.__class__.objects.filter(pk=pk).update(prefix="")
+
+
+def update_chant_incipit_field(chant: Chant):
```

Comment:
Type annotation for return? Functions with no explicit `return` return `None` I believe

Reply:
thanks for catching this!